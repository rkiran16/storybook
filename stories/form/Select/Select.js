import { html } from "lit-html";

export const Select = () => {
    return html`
    <form class="d-flex align-items-center">
        <label class="my-1 me-2" for="inlineFormCustomSelectPref">Select label</label>
        <select class="form-select w-25"  id="inlineFormCustomSelectPref">
        <option value="1">10MG</option>
        <option value="2">20MG</option>
        <option value="3">30MG</option>
        </select>
    </form>
    `;
}