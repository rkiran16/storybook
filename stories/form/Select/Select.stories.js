import { Select } from './Select';

export default {
   title: "Forms/Select"
}

const SelectTemplate = (args) => Select(args);

export const Default = SelectTemplate.bind({});