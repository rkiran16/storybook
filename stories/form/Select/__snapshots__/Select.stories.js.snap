// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Forms/Select Default 1`] = `
Object {
  "_$litType$": 1,
  "strings": Array [
    "
    <form class=\\"d-flex align-items-center\\">
        <label class=\\"my-1 me-2\\" for=\\"inlineFormCustomSelectPref\\">Select label</label>
        <select class=\\"form-select w-25\\"  id=\\"inlineFormCustomSelectPref\\">
        <option value=\\"1\\">10MG</option>
        <option value=\\"2\\">20MG</option>
        <option value=\\"3\\">30MG</option>
        </select>
    </form>
    ",
  ],
  "values": Array [],
}
`;
